src/legacy/core_plugins/data/public/search/aggs/metrics/median.ts

```diff
@@ -43,17 +43,13 @@ export const medianMetricAgg = new MetricAggType({
       name: 'field',
       type: 'field',
       filterFieldTypes: [KBN_FIELD_TYPES.NUMBER, KBN_FIELD_TYPES.DATE],
-    },
-    {
-      name: 'percents',
-      default: [50],
-    },
-    {
       write(agg, output) {
-        output.params.keyed = false;
+        output.params.field = agg.getParam('field').name;
+        output.params.percents = [50];
       },
     },
   ],
-  getResponseAggs: percentilesMetricAgg.getResponseAggs,
```

Comment:
is this not needed ?

Reply:
It's only needed for multi-value aggs. Replaced by `getValue`.